packages/marketplace/sst.config.ts

```diff
@@ -0,0 +1,52 @@
+/// <reference path="./.sst/platform/config.d.ts" />
+
+const PROJECT_NAME = "opruaas-marketplace";
+const REGION = "us-east-1";
+
+export default $config({
+  app(input) {
+    return {
+      name: PROJECT_NAME,
+      removal: input?.stage === "production" ? "retain" : "remove",
+      home: "aws",
+      providers: {
+        aws: {
+          region: REGION,
+          defaultTags: {
+            tags: {
+              customer: "op-ruaas",
+            },
+          },
+        },
+      },
+    };
+  },
+  async run() {
+
+    // static website
+    const ui = new sst.aws.StaticSite(`${PROJECT_NAME}-ui`, {
+      path: "packages/marketplace/ui",
```

Comment:
path should be `ui` only as we should run this from `marketplace` root. Also I think you should install `sst` in the `marketplace` root, I don't see any pakcage.json